artifacts/compatibility-yaml/compatibility-v0.7.0.yaml

```diff
@@ -0,0 +1,153 @@
+{
+  "CSI" : {
+    "2.7.1" : {
```

Comment:
This should be named 2.7.0 since we're using the deployment yamls for 2.7.0.
and can we have a parity for the min and max k8s version with this document? :) 

https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-D4AAD99E-9128-40CE-B89C-AD451DA8379D.html

Reply:
Updated the yamls